web/packages/teleterm/src/ui/Search/pickers/ActionPicker.tsx

```diff
@@ -239,14 +246,20 @@ const ExtraTopComponents = (props: {
   status: ActionPickerStatus;
   getClusterName: (resourceUri: uri.ClusterOrResourceUri) => string;
   showErrorsInModal: (errors: ResourceSearchError[]) => void;
+  advancedSearch: AdvancedSearch;
 }) => {
   const { status, getClusterName, showErrorsInModal } = props;
 
   switch (status.inputState) {
     case 'no-input': {
       switch (status.searchMode.kind) {
         case 'no-search': {
-          return <TypeToSearchItem hasNoRemainingFilterActions={false} />;
+          return (
+            <TypeToSearchItem
+              hasNoRemainingFilterActions={false}
+              advancedSearch={props.advancedSearch}
```

Comment:
nit: we de-structured this above, so we can just pass in `advancedSearch` instead of `props.advancedSearch` and in the cases below. Unless I'm missing a difference between the two?

Reply:
I think you mean a different component, but I added destructuring this prop.